csharp/src/Google.Protobuf/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
+    "NuSpec.ReferenceGenerator": { "type": "build", "version": "2.0.0-beta-bld14" }
+  },
+  "frameworks": {
+    "netstandard1.0": {}
```

Comment:
I am not trying to suggest we have to choose between .NET Core and Xamarin.

What I did for gRPC:

I kept the old .csproj project and old .sln file - that one will continue building with mono and all other legacy tools until we decide it's worth dropping it (perhaps in ~6months to a year).

I added a pure project.json solution that builds a nuget that explicitly targets both net45 and netstandard1.5. That one is buildable on linux, mac and windows (after instaling dotnet sdk) - which makes testing reasonable easy (and you don't depend on VS2015).

The new nuget file should be interchangeable with the old one (it adds netstandard support to what was already there). I'll probably publish the new dotnet as a preview/dogfood version and if things go well, it will become the official gRPC nuget package.